docs/source/transactions.md

```diff
@@ -458,16 +458,10 @@ Adds a new node to the pool or updates an existing node in the pool
 
 - `dest` (base58-encoded string):
 
-    Target Node's DID as base58-encoded string for 16 or 32 byte DID value.
+    Target Node's verkey as base58-encoded string for 16 or 32 byte DID value.
```

Comment:
Yes,  you are right. abbreviated verkeys are currently not supported. Can you please also fix the documentation for NODE request in `requests.md`?

Reply:
Hey! Just updated the same info in both `requests.md` and `requests-new.md`.

Reply:
Thank you very much!